securedrop/management/sources.py

```diff
@@ -0,0 +1,58 @@
+import argparse
+from typing import List
+
+from db import db
+from encryption import EncryptionManager, GpgKeyNotFoundError
+from management import app_context
+from models import Source
+
+
+def remove_pending_sources(args: argparse.Namespace) -> int:
+    """
+    Removes pending source accounts, with the option of keeping
+    the n newest source accounts.
+    """
+    n = args.keep_most_recent
+    sources = find_pending_sources(n)
+    print(f"Found {len(sources)} pending sources")
+
+    deleted = []
+    for source in sources:
+        try:
+            EncryptionManager.get_default().delete_source_key_pair(source.filesystem_id)
+        except GpgKeyNotFoundError:
+            pass
+        delete_pending_source(source)
+        deleted.append(source)
```

Comment:
`deleted` is never used? Maybe it was supposed to be used below in the message being printed, but if we just need to know the count, we could just increment a counter instead of holding onto the entire source object?